package.json

```diff
@@ -52,7 +52,7 @@
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
     "@bull-board/api": "3.6.0",
-    "@bull-board/express": "3.6.0",
+    "@bull-board/express": "3.7.0",
```

Comment:
You've got changes in here that don't belong.  Please rebase and squash your branch on `master`:

```
git checkout upstream
git pull upstream master
git checkout issue-2421-post-users-avatar
git rebase -i master
# fix conflicts if any and finish rebase...
git push origin -f issue-2421-post-users-avatar
```

Reply:
checkout upstream was not working... but i believe it's gone now.